src/AutoMapper/Configuration/MappingExpression.cs

```diff
@@ -127,6 +127,42 @@ public void ResolveUsing(Type valueResolverType, string memberName)
 
                 PropertyMapActions.Add(pm => pm.ValueResolverConfig = config);
             }
+
+            public void ConvertUsing(Type valueConverterType)
+            {
+                PropertyMapActions.Add(pm =>
+                {
+                    var config = new ValueConverterConfiguration(valueConverterType, valueConverterType.GetGenericInterface(typeof(IValueConverter<,>)));

```

Comment:
Not much reuse, but you can add another constructor for these two cases.

Reply:
I figured that the source member/names are optional so we'd need like 6 ctors to get every permutation

Reply:
I was thinking one extra constructor that takes only the converter type. Nitpicking.